packages/cli/src/ceramic-daemon.ts

```diff
@@ -258,9 +258,9 @@ export class CeramicDaemon {
     }
 
     const ceramic = new Ceramic(modules, params)
-    await ceramic._init(true)
     const did = new DID({ resolver: makeResolvers(ceramic, ceramicConfig, opts) })
-    await ceramic.setDID(did)
+    ceramic.did = did
```

Comment:
yes, this is necessarily temporarily until we stop indexing models from the config file, because now part of Ceramic.init() involves loading the Models, which requires a DID instance for the resolvers

Reply:
Makes total sense.